### ISSUE TITLE
Initial support for compiling with -std=c++14 which requires working around a bug in GNU libstdc++/libc [phab:D279]

### ISSUE THREAD

**Issue by kythe** [google/kythe#498]
_Created by **shahms** at 2015-05-22 23:28:06_: 

Workaround a GNU libstdc++/libc incompatibility with C++14

Add copyright header to c++config.h work around header

**Comment by kythe** [comment]
_**zarko** wrote at 2015-05-22 23:36:39_:

Tested this by changing the relevant cxx_flag to c++14 in CROSSTOOL.in, then:

```
CLANG=/usr/bin/clang-3.6 ./setup_bazel.sh
bazel clean --expunge
bazel test //...
```

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2015-05-22 23:36:39_

**Comment by kythe** [comment]
_**shahms** closed via commit ad51663b41e8d9d089681ea8d216d05528b85acb at 2015-05-22 23:47:59_
